docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -67,6 +67,49 @@ $ mvn deploy -Papache-release -DskipTests -Denforcer.skip -Dquarkus.build.skip -
 * This will close the repository from future deployments and make it available for others to view
 * After that, a popup window is created, enter the name and version of the artifact being released in the `Description` field and finally click `Close`
 
+== Collect the performance regression report
+
+Once the staging repository has been closed, run the `perf-regression` tool in order to compare the current staging against the previous released version.
+
+It's important that the machine running the `perf-regression` tool be equally loaded during the run.
+The simplest way would be to take a 1 hour slot at the end of the day, launch the tool on your main station and then lock the screen.
+
+For instance, running the tool to compare the `2.10.0` staging release against the `2.9.0` previous release would look as below:
+
+[source,shell]
+----
+$ cd tooling/perf-regression
+$ java -jar target/quarkus-app/quarkus-run.jar -an 2.9.0 2.10.0
+----
+
+After approximatively 1 hour, it should generate a report like beneath:
```

Comment:
Nitpick:

```suggestion
After approximatively 1 hour, it should generate a report like below:
```

Reply:
Subtle, so beneath is more a preposition. Many thanks for reporting :+1: